### ISSUE TITLE
Case of nicknames should be preserved

### ISSUE THREAD

**Issue by grifferz** [grifferz/pah-irc#1]
Nicknames are currently stored lower-cased:

```
<Paul> AgainstHumanity: me
<AgainstHumanity> paul: Nice! You're in!
<AgainstHumanity> We've now got 2 of minimum 4. Anyone else?
<AgainstHumanity> Type "AgainstHumanity: me" if you'd like to play too.
<grifferz> hmm, lower-cased your nick
```

It should be possible to store them in natural case and downcase only for comparisons.

**Comment by grifferz** [comment]
Ones I've fixed have a checked box:
- [x] e4b28e1 `<AgainstHumanity>` The Card Tsar is **filbert**; current Black Card:
- [x] 0084918 `<AgainstHumanity>` Waiting on **daviey** to make their play. Round started about **19h17m ago**. Idle punishment in about **4h43m**.
- [x] b5790f0
  - `<Alice>` AgainstDev: me
  - `<AgainstDev>` alice: Heyyy, you're already playing!
- [x] 1111b80
  - `<AgainstHumanity>` I'm forcibly resigning **daviey** from the game due to idleness. Idle since **2015-02-26T12:04:58**.
  - `<AgainstHumanity>` Now that **daviey** was dealt out, all the plays are in. No more changes!

**Comment by domcleal** [comment]
- [x] 1b6c204 Scores command outputs lower case names:
  - `<Dominic>` #redacted scores
  - `<AgainstHumanity>` [**#redacted**] Active Players: dominic(5) filbert(5) kfullert(3) chrisjrob(3) kwyjibo(2) eckrall(2) takumo daviey prando ging
  - `<AgainstHumanity>` [**#redacted**] Top 3 all time: dominic(5) filbert(5) kfullert(3) chrisjrob(3) kwyjibo(2) eckrall(2)
- [x] fc85f81 Also the black command outputs the tsar's name in lower case:
  - `<Dominic>` #redacted black
  - `<AgainstHumanity>` Current Black Card for game in #redacted:
  - `<AgainstHumanity>` → **Today on The Jeremy Kyle Show: "Help! My son is _____!"**
  - `<AgainstHumanity>` The current Card Tsar is **filbert**
  - `<AgainstHumanity>` Use the "Play" command to make your play!
- [x] b5790f0 d6cd59a Two instances here:
  - `<Dominic>` AgainstHumanity: winner 1
  - `<AgainstHumanity>` dominic: Sorry, you're not the Card Tsar – that's **filbert**.
- [x] 68f09d2 It looks like build_waitstring may produce lower case nicknames in the list of people, or single person that the game is waiting on.
- [x] a3c0c31 `<AgainstHumanity>` filbert: Which is the best play?
- [x] 72292fa `< AgainstHumanity>` The winner is **takumo**, who now has 1 Awesome Point! The new Card Tsar is **takumo**. Time for the next Black Card:
- [x] c1e9511 Private message: `<AgainstHumanity>` [#redacted] The winner was **filbert**, who now has **6** Awesome Points! Their winning play was:

**Comment by domcleal** [comment]
Also the black command outputs the tsar's name in lower case:

<pre>
213721 &lt;Dominic&gt; #redacted black
213721 &lt;AgainstHumanity&gt; Current Black Card for game in #redacted:
213721 &lt;AgainstHumanity&gt; → Today on The Jeremy Kyle Show: "Help! My son is _____!"
213722 &lt;AgainstHumanity&gt; The current Card Tsar is filbert
213722 &lt;AgainstHumanity&gt; Use the "Play" command to make your play!
</pre>

**Comment by domcleal** [comment]
Two instances here:

<pre>
214222 &lt;@Dominic&gt; AgainstHumanity: winner 1
214222 &lt; AgainstHumanity&gt; dominic: Sorry, you're not the Card Tsar – that's filbert.
</pre>

It looks like build_waitstring may produce lower case nicknames in the list of people, or single person that the game is waiting on.

**Comment by grifferz** [comment]
Thanks @domcleal. I've consolidated all your reports into one comment and will check them off as they're fixed.

**Comment by grifferz** [comment]
* [ ]
    * `<AgainstHumanity>` I'm forcibly resigning **daviey** from the game due to idleness. Idle since **2015-02-26T12:04:58**.
    * `<AgainstHumanity>` Now that **daviey** was dealt out, all the plays are in. No more changes!

**Comment by domcleal** [comment]
Thanks, I added more to https://github.com/grifferz/pah-irc/issues/1#issuecomment-76476364.

**Comment by domcleal** [comment]
- [x] 515a41d Only one instance was fixed in 72292fa, there's still the Tsar name:
  - `<AgainstHumanity>` The winner is **Filbert**, who now has 2 Awesome Points! The new Card Tsar is **dominic**. Time for the next Black Card:

**Comment by domcleal** [comment]
I think you've probably fixed this issue with the latest round of commits, thanks!

**Comment by domcleal** [comment]
- [x] b7d2a9a When a channel name is given in an unknown command privately, the sender's name is returned in lower case.  (Seems to always be downcased here: https://github.com/grifferz/pah-irc/blob/515a41d/lib/PAH.pm#L536)
  - `<Dominic>` #issue 1
  - `<AgainstHumanity>` dominic: Sorry, that's not a command I recognise. See https://github.com/grifferz/pah-irc#usage for more info.

**Comment by grifferz** [comment]
Calling this one done now, then.

**Comment by domcleal** [comment]
Always one more...

* [ ] `<+AgainstHumanity>` Daviey: Sorry, this hand is complete and we're waiting on **dominic** to pick the winner. Please try again later.
